.circleci/config.yml

```diff
@@ -38,7 +38,7 @@ jobs: # a collection of steps
       # do the actual build
       - run:
           command: mvn clean install --fae
-          no_output_timeout: 20m
+          no_output_timeout: 1h
```

Comment:
why this?

Reply:
CI fails due to 'Too long with no output (exceeded 20m0s): context deadline exceeded' (see https://app.circleci.com/pipelines/github/EMResearch/EvoMaster/2078/workflows/4dc9592d-fdc8-4757-8999-dd2add437c31/jobs/2149)

Reply:
and? if a test hangs, then build should fail. moving from 20m to 60m just makes it fail 40m later...
do we have any case of a test that would run for more than 20m?

Reply:
I changed this back, let's see whether it will fail.